### ISSUE TITLE
Hyperparameters for training

### ISSUE THREAD

**Issue by andriyrizhiy** [nowsyn/SIM#9]
HI! Good work! I have a question about hyperparameters. Wich optimizer and learning rate policy do you use for the best model?

**Comment by zhuzhu18** [comment]
训练代码没有开源

**Comment by nowsyn** [comment]
We will release the training code and the updated model in one week.
